BurnerB/PropertyProLite-CH2
# Ch folder structure 166856512

#### What does this PR do?
Structure the app and add dependancies
#### Description of Task to be completed?
* Structure the app according to best practices
* configure babel transpiler
* configure eslint linter
#### How should this be manually tested?
CLONE this repo
CD into it
Navigate to the folder where you cloned the repo.
open PropertyProLite-CH2 with your favorite editor and observe the folder structure
run `npm start` to ensure the app is working
#### What are the relevant pivotal tracker stories?
#166856512
#### Screenshots (if appropriate)
